tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/effects/BUILD

```diff
@@ -0,0 +1,25 @@
+package(default_visibility = ["//tensorboard:internal"])
+
+load("//tensorboard/defs:defs.bzl", "tf_ts_library")
+load("@npm_angular_bazel//:index.bzl", "ng_module")
+
+ng_module(
+    name = "effects",
+    srcs = [
+        "debugger_effects.ts",
+        "index.ts",
+    ],
+    deps = [
+        "//tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/actions",
+        "//tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/data_source",
+        "//tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store",
+        "@npm//@angular/common",
+        "@npm//@angular/core",
+        "@npm//@ngrx/effects",
+        "@npm//@ngrx/store",
+        "@npm//rxjs",
+    ],
+)
+
+# TODO(cais): After TestBed.inject() is figured out in OSS, add
```

Comment:
Can you explain here in GitHub comments what this would mean? You seem to have written a good test with TestBed.get which is more or less equal to inject